rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -63,8 +64,12 @@ void SqliteStorage::open(
       throw std::runtime_error("Failed to read from bag '" + uri + "': No metadata found.");
```

Comment:
:warning: the block below checks if metadata exists. If this is the case, it is set (`database_name_ = metadata->relative_file_paths[0];`) and then `database_name` is being overwritten on line 67 - is this expected?